reana_commons/config.py

```diff
@@ -81,3 +81,12 @@
                                     '%(levelname)s - '
                                     '%(message)s')
 """Format of engine logs."""
+
+CVMFS_REPOSITORIES = {
+    'alice.cern.ch': 'alice',
+    'atlas.cern.ch': 'atlas',
+    'cms.cern.ch': 'cms',
+    'fcc.cern.ch': 'fcc',
+    'lhcb.cern.ch': 'lhcb',
```

Comment:
If a user would like to include others, e.g. `atlas-conddb`, would this require to release a new version of `r-commons`?

If so, then we may add already some used ones, recall that yadage example:

- atlas-condb.cern.ch
- lhcb-condb.cern.ch 
- sft.cern.ch

Reply:
Yes, any repository to be mounted must be included in the `CVMFS_REPOSITORIES` config map, as we need a way of identifying it.
Added all repositories present in HPC/SLURM.